tests/dropped_frame_handling.rs

```diff
@@ -19,7 +19,7 @@ async fn handle_dropped_frames_gracefully_async() {
         .unwrap();
 
     // max_num_pending_frames is one!
-    let mut profiler = wgpu_profiler::GpuProfiler::new(1, queue.get_timestamp_period(), device.features());
+    let mut profiler = wgpu_profiler::GpuProfiler::new(&adapter, &device, &queue, 4);
```

Comment:
I think this test specifically uses `1` here so that a frame will be "dropped" below.